### ISSUE TITLE
Sending object detected info to a TCP/IP client 

### ISSUE THREAD

**Issue by ngingihy** [AlexeyAB/darknet#3166]
I made a TCP/IP client that receives the JSON string from port 8070 but for some reason, I'm only receiving the first frame and then nothing gets sent from the darknet server and also the string sent has HTTP weird information like this 

>  Received b'HTTP/1.0 200 OK\r\nServer: Mozarella/2.2\r\nAccept-Range: bytes\r\nConnection: close\r\nMax-Age: 0\r\nExpires: 0\r\nCache-Control: no-cache, private\r\nPragma: no-cache\r\nContent-Type: application/json\r\n\r\n[\n{\n "frame_id":2, \n "objects": [ \n\n ] \n}'".

**My goal is to notify the client once the confidence level is less than 20% so my camera can automatically zoom in.**  

My client code is below

`I made a TCP/IP client that receives the JSON string from port 8070 but for some reason, I'm only receiving the first frame and then nothing gets sent from the darknet server and also the string sent has HTTP weird information like this 

>  Received b'HTTP/1.0 200 OK\r\nServer: Mozarella/2.2\r\nAccept-Range: bytes\r\nConnection: close\r\nMax-Age: 0\r\nExpires: 0\r\nCache-Control: no-cache, private\r\nPragma: no-cache\r\nContent-Type: application/json\r\n\r\n[\n{\n "frame_id":2, \n "objects": [ \n\n ] \n}'".

**My goal is to notify the client once the confidence level is less than 20% so my camera can automatically zoom in.**  

My client code is below

`#!/usr/bin/env python3

import socket

HOST = '127.0.0.1'  # The server's hostname or IP address
PORT =8070      # The port used by the server

with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
	s.connect((HOST, PORT))
    #s.sendall('Hello, world')
	while True:
		data = s.recv(1024)
		print('Received', repr(data))`

**Comment by AlexeyAB** [comment]
> I made a TCP/IP client that receives the JSON string from port 8070 but for some reason, I'm only **receiving the first frame and then nothing gets sent from the darknet server** and also the string sent has HTTP weird information like this


Just try to connect to darknet by using Chrome or Firefox web-browser `http://ip:8070`, do you get requried results?


I will add flag to disable HTTP-head, currently just ignore it in your program.

**Comment by ngingihy** [comment]
Yes, I get the required results, however, I want to see this result on a client socket, not web-browser. I also want to all the frames' JSON strings get sent but using the client code above I only receive one frame.

**Comment by AlexeyAB** [comment]
If you get info in Web-browser for all frames, then the issue is in your client-code.

Disable HTTP-head just comment these lines: https://github.com/AlexeyAB/darknet/blob/cce34712f6928495f1fbc5d69332162fc23491b9/src/http_stream.cpp#L215-L226

**Comment by HuanzhaoSong** [comment]
> > I made a TCP/IP client that receives the JSON string from port 8070 but for some reason, I'm only **receiving the first frame and then nothing gets sent from the darknet server** and also the string sent has HTTP weird information like this
> 
> Just try to connect to darknet by using Chrome or Firefox web-browser `http://ip:8070`, do you get requried results?
> 
> I will add flag to disable HTTP-head, currently just ignore it in your program.

what should i do if it's Internet Explorer, always prompt to download ? thanks

**Comment by VirtuallKingg** [comment]
@ngingihy I am also trying to do so similar kind of thing. But I do not have any idea. Can you please describe me how did you do it? I want to send the livestream detection result via tcp/ip. And, I do not have any idea how to do and which file to make changes. Please can you describe it in stepwise and in details. Thanks in advance.

**Comment by PIjarihd** [comment]
@ngingihy  I am also trying to do so similar kind of thing. But I do not have any idea. Can you please describe me how did you do it? I want to send the livestream detection result via tcp/ip. And, I do not have any idea how to do and which file to make changes. Please can you describe it in stepwise and in details. Thanks in advance

**Comment by iraadit** [comment]
Hi @ngingihy 
Did you find how to continue to receive the json output for each frame?
Thanks
